manifests/init.pp

```diff
@@ -40,37 +56,33 @@
 #
 # Phil Fenstermacher <phillip.fenstermacher@gmail.com>
 #
-# lint:ignore:variable_is_lowercase
 class ca_cert (
-  Boolean $always_update_certs = false,
-  Boolean $purge_unmanaged_CAs = false, # lint:ignore:variable_contains_upcase
-  Boolean $install_package     = true,
-  Boolean $force_enable        = false,
-  Hash    $ca_certs            = {},
-  String  $package_ensure      = 'installed',
-  String  $package_name        = $ca_cert::params::package_name,
-) inherits ca_cert::params {
-  include ca_cert::params
-  include ca_cert::update
-
-  if $always_update_certs == true {
-    Exec <| title=='ca_cert_update' |> {
-      refreshonly => false,
-    }
+  Boolean $always_update_certs             = false,
+  Boolean $purge_unmanaged_CAs             = false, # lint:ignore:variable_is_lowercase lint:ignore:variable_contains_upcase
+  Boolean $install_package                 = true,
+  Boolean $force_enable                    = false,
+  Hash    $ca_certs                        = {},
+  String  $package_ensure                  = 'installed',
+  String[1] $package_name                  = 'ca-certificates',
+  String[1] $trusted_cert_dir              = '/etc/pki/ca-trust/source/anchors',
+  Optional[String[1]] $distrusted_cert_dir = undef,
+  String[1] $update_cmd                    = 'update-ca-trust extract',
+  String[1] $cert_dir_group                = 'root',
+  String[1] $cert_dir_mode                 = '0755',
+  Boolean $supported                       = false,
```

Comment:
Not sure I like this `supported` parameter pattern. I don't think we do this in any of the Vox Pupuli modules. It seems redundant to just listing supported OSes in `metadata.json`. I suppose you are retaining existing behavior.

Reply:
Yes this is only here to maintain the existing behaviour. Would be more than happy to remove it :)
Let @pcfens decide which route to go.

Reply:
I do agree with @kenyon on this one. @pcfens doesn't seem to have any opinion?